<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeAutoBuild</class>
 <widget class="QMainWindow" name="CodeAutoBuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeAutoBuild</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLineEdit" name="SuiteLineEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="DisplayLineEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="runpushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>一键生成可执行程序</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="chooseDisplay">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>选择Display文件夹</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/openfold.png</normaloff>:/images/openfold.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="planecomboBox">
        <item>
         <property name="text">
          <string>机型选择</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F-35</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F-22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F-18</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1" rowspan="2">
       <widget class="QPushButton" name="vsDisplaybtn">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>选择VS工程Display文件夹</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/openfold.png</normaloff>:/images/openfold.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLineEdit" name="vsDisplayEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QLineEdit" name="vsSuiteEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QPushButton" name="chooseSuite">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>选择Suite文件夹</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/openfold.png</normaloff>:/images/openfold.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="vsSuitebtn">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>选择VS工程Suite文件夹</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/openfold.png</normaloff>:/images/openfold.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="progressText">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>程序进行中：</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyProgressBar" name="progresswidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyProgressBar</class>
   <extends>QWidget</extends>
   <header>myprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
